extensions/resteasy-reactive/quarkus-rest-client-microprofile/runtime/src/main/java/io/quarkus/rest/rest/client/microprofile/MicroProfileRestClientResponseFilter.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.rest.rest.client.microprofile;
+
+import java.io.IOException;
+import java.util.List;
+
+import javax.ws.rs.ProcessingException;
+import javax.ws.rs.client.ClientRequestContext;
+import javax.ws.rs.client.ClientResponseContext;
+import javax.ws.rs.client.ClientResponseFilter;
+
+import org.eclipse.microprofile.rest.client.ext.ResponseExceptionMapper;
+import org.jboss.resteasy.reactive.client.handlers.ClientResponseRestHandler;
+import org.jboss.resteasy.reactive.client.impl.ClientRequestContextImpl;
+import org.jboss.resteasy.reactive.client.impl.ClientResponseContextImpl;
+import org.jboss.resteasy.reactive.common.jaxrs.ResponseImpl;
+
+public class MicroProfileRestClientResponseFilter implements ClientResponseFilter {
+    private final List<ResponseExceptionMapper<?>> exceptionMappers;
+
+    public MicroProfileRestClientResponseFilter(List<ResponseExceptionMapper<?>> exceptionMappers) {
+        this.exceptionMappers = exceptionMappers;
+    }
+
+    // mstodo aint' registered!
+    @Override
+    public void filter(ClientRequestContext requestContext, ClientResponseContext responseContext) throws IOException {
+        if (exceptionMappers != null) {
```

Comment:
Maybe move the `null` check to constructor and blow up if it happens? This filter doesn't seem to make a lot of sense without any exception mappers.

Reply:
actually, exceptionMappers are never null in the current impl